src/chaplin/lib/route.coffee

```diff
@@ -38,6 +38,8 @@ module.exports = class Route
     # Clone options.
     @options = if options then _.extend({}, options) else {}
 
+    @options.paramsInQS = true if @options.paramsInQS isnt false
```

Comment:
@akre54 I'm late to the game, but why did you chose an opt-out for this behaviour? This is a breaking change.